internal/migrations/sql.go

```diff
@@ -51,16 +51,15 @@ var (
 	tableMigrationsSQL = `CREATE TABLE IF NOT EXISTS ` + migrationsSchemaName + `.migration (
 		id INT PRIMARY KEY,
 		start_height INT8 NOT NULL,
-		end_height INT8 NOT NULL,
-		chain_id TEXT NOT NULL
+		end_height INT8 NOT NULL
 	)`
 
 	// getMigrationSQL is the sql query used to get the current migration.
-	getMigrationSQL = `SELECT start_height, end_height, chain_id FROM ` + migrationsSchemaName + `.migration;`
+	getMigrationSQL = `SELECT start_height, end_height FROM ` + migrationsSchemaName + `.migration;`
 	// migrationIsActiveSQL is the sql query used  to check if a migration is active.
 	migrationIsActiveSQL = `SELECT EXISTS(SELECT 1 FROM ` + migrationsSchemaName + `.migration);`
 	// createMigrationSQL is the sql query used to create a new migration.
-	createMigrationSQL = `INSERT INTO ` + migrationsSchemaName + `.migration (id, start_height, end_height, chain_id) VALUES ($1, $2, $3, $4);`
+	createMigrationSQL = `INSERT INTO ` + migrationsSchemaName + `.migration (id, start_height, end_height) VALUES ($1, $2, $3, $4);`
```

Comment:
`createMigrationSQL = `INSERT INTO ` + migrationsSchemaName + `.migration (id, start_height, end_height) VALUES ($1, $2, $3);`

should take only 3 arguments here instead of 4

Reply:
Ahh thank you